go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/cenkalti/backoff/v4 v4.2.1 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/ethereum/go-ethereum v1.10.26 // indirect
```

Comment:
:wastebasket:

Reply:
hmm.

```
This module is necessary because [github.com/ethereum/go-ethereum/common](vscode-file://vscode-app/Applications/Visual%20Studio%20Code%202.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) is imported in:

- github.com/smartcontractkit/chainlink-common/pkg/types/mercury/v2
-- github.com/smartcontractkit/libocr/offchainreporting/types
```

this comes from v2/DataSource, which was copy and pasted from core.

what is the right way to manage this? change it in both places to use ocr2 types? i believe the references are to Report and ReportTimestamp, which i also bellieve are the same underlaying types in both ocr and ocr2.

is it safe to make that change? it feels dangerous to me (i.e open to subtle bugs if some nops are running different versions of `v2` that have different types)

Reply:
Can you link to the types you are referencing? I cannot find them.

Reply:
I think I found it. Are you sure that is referencing the correct `ReportTimestamp`? It seems like a mistake.

Reply:
I think you just need to do `offchainreporting/types` -> `offchainreporting2plus/types`

Reply:
https://github.com/smartcontractkit/libocr/blob/6359502d2ff1165c7e6b77b9eff2c5a46a7a4fbb/offchainreporting/types/db.go#L26

in that same pkg https://github.com/smartcontractkit/libocr/blob/6359502d2ff1165c7e6b77b9eff2c5a46a7a4fbb/offchainreporting/types/types.go#L11

Reply:
ignore. it looks like i copied the wrong thing or otherwise screwed up the pkg import here
https://github.com/smartcontractkit/chainlink-data-streams/blob/a6e3fe8ff2a12886b111f341639bae3cbf478501/mercury/v2/mercury.go#L15C2-L15C67

fixed now